mapr/datadog_checks/mapr/mapr.py

```diff
@@ -181,7 +182,14 @@ def should_collect_metric(self, metric_name):
 
     def submit_metric(self, metric):
         metric_name = metric['metric']
-        tags = self.base_tags + ["{}:{}".format(k, v) for k, v in iteritems(metric['tags'])]
+        tags = copy.deepcopy(self.custom_tags)
+        for k, v in iteritems(metric['tags']):
+            if self._disable_legacy_cluster_tag:
+                if k == 'clustername':
+                    k = 'mapr_cluster'
+                elif k == 'clusterid':
+                    k = 'mapr_cluster_id'
```

Comment:
We should do like for other integrations, and send the new tag anyway

* If `disable_legay_cluster_tag==False`:
   send `clustername`, `mapr_cluster`, `clusterid`, `mapr_cluster_id`
*  If `disable_legay_cluster_tag==True`:
  send `mapr_cluster` and `mapr_cluster_id` only

Reply:
Isnt it what is happening here? `if self._disable_legacy_cluster_tag` then I replace the keys otherwise I leave them as they are